src/test_helpers/integration.cljs

```diff
@@ -182,6 +182,29 @@
                         (restore-fn)
                         (done))))))))
 
+(defn test-app-initialization
```

Comment:
I extracted these 2 tests here because they are identical for contract & integration tests.

Reply:
🙌